python/ingestion/standardized_columns.py

```diff
@@ -122,6 +122,12 @@
 JAIL_PREFIX = "jail"
 INCARCERATED_PREFIX = "incarcerated"
 
+# HIV
+HIV_CASES = 'hiv_cases'
```

Comment:
let's not call this cases, and instead keep the `diagnoses` terminology. since when we expand HIV to include other data types, it'll be confusing what the generic `cases` is referring to